### PR TITLE
[jackett] update jackett to version-v0.17.591

### PR DESCRIPTION
**Pull request by jmmaloney4**
Signed-off-by: Jack Maloney <jmmaloney4@gmail.com>

**Description of the change**

Update image for jackett chart to `version-v0.17.591`.

**Benefits**

**Possible drawbacks**

**Applicable issues**

- fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] (optional) Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [x] (optional) Variables are documented in the README.md

### CODE REVIEW

File: charts/jackett/values.yaml [hunk 1]
```diff
@@ -17,6 +17,8 @@ env: {}
   # TZ: UTC
   # PUID: 1001
   # PGID: 1001
+  ## Allow Jackett to update inside of the container, but might cause outages if turned on.
+  # AUTOUPDATE: false
```
Comment (bjw-s) [inline charts/jackett/values.yaml:21]:
```suggestion
```
This is not really the way containers should run software updates. I don't think we should explicitly mention or support this.

**Review by bjw-s — CHANGES_REQUESTED** [review]
Looks good for the most part 👍 Just a small change requested 😄

File: charts/jackett/Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: v0.16.2106
+appVersion: 0.17.591
 description: API Support for your favorite torrent trackers
 name: jackett
-version: 7.0.1
+version: 7.1.0
```
Comment (bjw-s) [inline charts/jackett/Chart.yaml:5]:
```suggestion
version: 7.0.2
```
I would be fine with only bumping the patch version for this

File: charts/jackett/values.yaml (hunk 1 above)
Comment (onedr0p) [reply charts/jackett/values.yaml:21]: This environment variable isn't doing anything anyways, I don't see it in the source of Jackett.

Comment (onedr0p) [reply charts/jackett/values.yaml:21]: It's part of LSIO image, which defaults to `AUTOUPDATE=true`... very odd

File: charts/jackett/Chart.yaml (hunk 2 above)
Comment (onedr0p) [reply charts/jackett/Chart.yaml:5]: Agreed, this is not a feature bump

File: charts/jackett/values.yaml (hunk 1 above)
Comment (jmmaloney4) [reply charts/jackett/values.yaml:21]: Yes, the default true was the only reason I added it in.

File: charts/jackett/Chart.yaml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: v0.16.2106
+appVersion: 0.17.591
```
Comment (onedr0p) [inline charts/jackett/Chart.yaml:2]:
```suggestion
appVersion: v0.17.591
```

**Pull request merged**